charts/portworx/v2.8.1/templates/_helpers.tpl

```diff
@@ -95,6 +95,35 @@ release: {{ .Release.Name | quote }}
 {{- end -}}
 {{- end -}}
 
+
+{{- define "px.getCSIProvisionerImage" -}}
+{{- if semverCompare "<=1.6-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
does this equate to k8s version of <= 1.16?

Reply:
i don't think so. i somehow confused myself with the versioning there. thanks for the catch.